tools/car_porting/README.md

```diff
@@ -64,3 +64,11 @@ An example of searching through a database of segments for a specific condition,
 ![steer warning example](https://github.com/commaai/openpilot/assets/9648890/d60ad120-4b44-4974-ac79-adc660fb8fe2)
 
 *a plot of the steer_warning vs steering angle, where we can see it is clearly caused by a large steering angle change*
+
+### [tools/car_porting/examples/subaru_long_accel.ipynb](/tools/car_porting/examples/subaru_long_accel.ipyn)
```

Comment:
why not use plotjuggler?

Reply:
it would allow you to view many segments at once, and also this is mostly just to give examples of parsing data